skyxplore-ui-test/src/test/java/com/mentoring/test/FriendRequestTest.java

```diff
@@ -102,4 +109,109 @@ public void sendAndReceiveFriendRequest() {
         softAssertion.assertAll();
     }
 
+    @Description("Negative test case 1: " +
+            "User A and user B exist. " +
+            "User A has already sent friend request to user B's character on the backend. " +
+            "User B logs in and refuses user A's request.")
+    @Test(groups = SMOKE)
+    @Severity(SeverityLevel.BLOCKER)
+    @Feature(FRIEND_REQUEST)
+    public void sendAndDeclineFriendRequest() {
+
+        userA.sendFriendRequest(characterIdA, accessTokenIdForUserA, userIdForUserA, characterIdB);
+
+        CommunityPage communityPage = new IndexPage(driver)
+                .login(userB)
+                .selectCharacter(characterNameB)
+                .openCommunityPage()
+                .clickFriendRequestsButton()
+                .clickReceivedFriendRequestDeclineButton(characterNameA);
+
+        NotificationContainer notificationContainer = new NotificationContainer(driver);
+
+        CommonAssertions.assertNotificationMessageIsCorrect(softAssertion, notificationContainer, Messages.DECLINED_FRIEND_REQUEST);
+
+        softAssertion.assertAll();
+    }
+
+    @Description("Negative test case 2: " +
+            "User B has a pending request from user A. " +
+            "User B renames the character he got the request with, " +
+            "then accepts the request.")
+    @Test(groups = SMOKE)
+    @Severity(SeverityLevel.BLOCKER)
```

Comment:
Blocker level is an exaggeration in my opinion, as it's not a general use case (rather edge-case). Please revisit the severity levels in this class (blocker=major functionality is not working at all, critical path cannot be completed successfully)